samples/bluetooth/scanning_while_connecting/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  description: Bluetooth Low Energy fast connection establishment sample
+  name: Bluetooth LE central scanning while connecting
+tests:
+  sample.bluetooth.scanning_while_connecting:
+    sysbuild: true
+    build_only: true
+    integration_platforms:
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Can you add `nrf54l15dk/nrf54l10/cpuapp` and `nrf54l15dk/nrf54l05/cpuapp` as well?